### ISSUE TITLE
Keep Alive

### ISSUE THREAD

**Issue by gabirosca** [h3imdall/ftp-remote-edit#406]
Hello, 
is there a possibility to implement keep alive option for connections?
Thank you.

**Comment by mcplayerteam** [comment]
Agreed. Currently after opening a remote connection, the connection no longer works after a variable amount of time (inactivity timeout), and files become inaccessible (it tries to connect to get the file without reestablishing connection, but fails after a timer). All I can do is close the connection (the folder with the connection name) and reestablish the connection (reopen the folder with the connection name), then everything works again, except that inactivity timeout would kick in again.
Could we have some way of keeping the connection active, or if it is a bad practice, then instead just reestablish the connection upon a new request?
